src/model/data_access_layer/users/users_reader.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Tuple
+
+from sqlalchemy.orm.query import Query
+
+from src.model.tables import Users
+
+
+class UsersReader:
+
+    def __init__(self, alchemist):
+        self.session = alchemist.session
+
+    def __get_list_from_raw_list(self,
```

Comment:
I see this method in lot of classes. You could just create a fucntion which will be used in your DAL